pdc_client/__init__.py

```diff
@@ -130,8 +130,10 @@ class to perform requests.
                         ssl_verify = False
                     elif not insecure and not ssl_verify:
                         ssl_verify = True
-            elif ssl_verify is True:
+            elif ca_cert_or_insecure is True:
                 ssl_verify = False
+            else:
+                ssl_verify = ca_cert_or_insecure
```

Comment:
If options.ca_cert is `none` and options.insecure is `false` then ssl_verify will be set to `false`. Is it because if no ca_cert then client will not use ssl_verify anyway, even insecure is set to false in command line?